internal/handler/command.go

```diff
@@ -22,6 +23,11 @@ import (
 	contract "github.com/edgexfoundry/go-mod-core-contracts/models"
 )
 
+const (
+	SDKRegexSyntax string    = "&ds-.*=.*&"
```

Comment:
This doesn't handle this case: 
     `some-endpoint&ds-something=false`
I will not be matched and would need to be replaced by `""`.
You are using `&` as a terminating character, but it gets included in the match. Is there a better way to match up to the end of a query parameter, i.e just the `&ds-something=false` in my example above?

Reply:
Will there be other non-word characters besides `-` `_` in the query parameter ? I'm thinking of change the regex to `&ds-[\w-]*=[^&]*` but need to add non-word characters into expression

Reply:
We can't restrict what names a device service developer uses, but I think we can make restrictions on the names the SDK uses which are these ds-<name>query parameters that we are trying to remove from passing onto the device service.  We are already restricting them to start with a specify prefix... We just need to document the naming restrictions. 

As I write this, I am think these SDK specific query parameters have the `sdk` or `ds-sdk` prefix rather than the `ds` prefix? The `ds` prefix to me indicates that they a for the Device Service, which they are not. @iain-anderson, @cloudxxx8, @tonyespy, @rsdmike , thoughts?

Reply:
I think that instead of calling the CommandHandler with req.URL.RawQuery, we should use req.URL.Query() instead (returns Values object). It should be relatively straightforward to filter out the "ds-" parameters from there, and we can reconstruct a raw query string from whatever remains. Then for the Geneva codebase we can change the API so that we pass the filtered Values object straight to the driver.

Reply:
@lenny-intel and @iain-anderson , Chris has modified this PR to meet your comment.  Please help review again.  Thanks.